packages/subscriptions/graphql/resolvers/_mutations/sendTestNotification.js

```diff
@@ -0,0 +1,46 @@
+const { ensureUser } = require('@orbiting/backend-modules-auth')
+
+module.exports = async (_, args, context) => {
+  const { submitComment } = require('@orbiting/backend-modules-discussions/lib/Notifications')
+  const { notifyPublish } = require('../../../../../servers/publikator/lib/Notifications')
+
+  const {
+    commentId,
+    docRepoId
```

Comment:
Just a thought, `repoId` would be enough as we handle no repository IDs but  those of documents.